jwst/cube_build/ifu_cube.py

```diff
@@ -2112,8 +2112,10 @@ def setup_final_ifucube_model(self, model_ref):
                 dq = dq[remove_start:self.naxis3 - remove_final, :, :]
                 var = var[remove_start:self.naxis3 - remove_final, :, :]
 
+                # update WCS information if removing wavelengths from the IFU Cube
                 if self.linear_wavelength:
                     self.crval3 = self.zcoord[remove_start]
+                    self.naxis3 = self.naxis3 - (remove_start + remove_final)
```

Comment:
I assume the arithmetic works out correctly here, i.e. you don't need to add or subtract 1 somewhere to account for inclusive start/end values?

Reply:
Ok good catch. Naxis3 needed +1.
So I got the regression tests to run using Jenkins. There were come failures - but in imaging modes - nothing to do with this PR. Do I just ignore those and move on.